pallets/moonbeam-lazy-migrations/src/benchmarks.rs

```diff
@@ -26,9 +26,9 @@ benchmarks! {
 	clear_suicided_storage {
 		let caller = account("caller", 1, 100);
 		// a is the number of addresses
-		let a in 0 .. GetArrayLimit::get();
+		let a in 1 .. GetArrayLimit::get() => 100;
 		// l is the limit of the number of storage entries to be deleted
-		let l in 0 .. 32500;
+		let l in 1 .. 32500 => 500;
```

Comment:
@librelois @RomarQ I fixed the benchmark problem by setting the minimum number of addresses and storage entries to 1 in the setup. This setup involves many iterations because of the variables and upper limit. It will take some time to run. I wonder if we can change the upper limit to have less iterations ?

Reply:
> I wonder if we can change the upper limit to have less iterations ?

Yes we can, we don't need to test all possible values, we just need enough points for the benchs to be able to induce a linear relationship

Reply:
I think we will never use a limit that high. We can introduce a new constant `CLEAR_SUICIDED_STORAGE_MAX_LIMIT = 1000` and validate it inside the extrinsic.